ui/selectors/alerts.ts

```diff
@@ -0,0 +1,27 @@
+import { createSelector } from 'reselect';
+import {
+  Alert,
+  ConfirmAlertsState,
+} from '../ducks/confirm-alerts/confirm-alerts';
+
+export type AlertsState = {
+  confirmAlerts: ConfirmAlertsState;
+};
+
+export function selectAlerts(state: AlertsState, ownerId: string): Alert[] {
+  return state.confirmAlerts.alerts[ownerId] || [];
```

Comment:
Should we use `??` here as we don't care if it's falsey, but only `null` or `undefined`?